### ISSUE TITLE
antd新增的两个less变量@white（#ffffff）和@black(#000000)导致generate出来的color.less错误

### ISSUE THREAD

**Issue by githubCK** [mzohaibqc/antd-theme-generator#14]

**Comment by mzohaibqc** [comment]
@githubCK Thanks for reporting this issue. I'll fix this soon

**Comment by mzohaibqc** [comment]
This is due to Ant Design `popover` style error 
https://github.com/ant-design/ant-design/issues/14148
https://github.com/ant-design/ant-design/blob/master/components/popover/style/index.less#L58
```
  &-inner {
    background-color: @popover-bg;
    background-clip: padding-box;
    border-radius: @border-radius-base;
    box-shadow: @box-shadow-base;
    box-shadow: ~'0 0 8px @{shadow-color} \9';
  }```

**Comment by mzohaibqc** [comment]
@githubCK Above mentioned issue has been fixed in `antd-theme-generator@1.1.6` so update your project dependency to fix this issue.
